mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -32,6 +31,12 @@ def __init__(self, name, config):
         self.baudrate = config.get('baudrate', DEFAULT_BAUD_RATE)
         self.image_path = config.get('image_path', None)
         self.polling_timeout = int(config.get('polling_timeout', 60))
+        self.allocate_requirements = { "platform_name": self.platform_name }
+
+        if(config.get('tags', None)):
```

Comment:
Could you drop the parenthesis here? it's more idiomatic in python to omit parenthesis for simple checks.